### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Samisafool**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
+3. Launch `DevLauncher`.
+4. Go to account settings, and you will be requested to use email and password.
+5. As your email use your username, and then type a random password or use ElyBy account.
+6. Save it.
+7. Now enjoy the Launcher.
+
+In case if you are using MacOS/Linux then additional step is to make a script `DevLauncher` executable by using the command `chmod +x DevLauncher` in the terminal
 
-In case if you are using MacOS/Linux then additional step is to make a script `MultiMC` executable by using the command `chmod +x MultiMC` in the terminal
+To remove the ugly "localhost:nnnn" or "ElyBy", add `-Dauthlibinjector.noShowServerName` to the java arguments.
```
Comment (maximmasterr) [inline README.md:20]: Please remove word ugly, be more polite to information

Comment (Samisafool) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,32 @@
-> **This is a "cracked" version of a popular Minecraft launcher that lets you play the game without a Mojang account.**
->
-> This software is not related to MultiMC developers and provided without any warranty. Please don't bomb MultiMC developers if something gets wrong using this launcher.
+# **This is a "cracked" version of a popular Minecraft launcher that lets you play the game without a Mojang account.**
+
+# This software is not related to MultiMC developers and provided without any warranty. Please don't bomb MultiMC developers if something gets wrong using this launcher.
```
Comment (maximmasterr) [inline README.md:3]: I think we can change this to second-title(`##`) or even third-title(`###`)

File: README.md [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
-1. Download pre-built binaries for your system
-2. Unpack them in your desired directory
-3. Launch the MultiMC
-4. Go to account settings, and you will be requested to use email and password
-5. As your email use your username, and then type a random password
-6. Save it
-7. Now enjoy MultiMC
+1. Download pre-built binaries for your system.
+2. Unpack them in your desired directory.
+3. Launch `DevLauncher`.
```
Comment (maximmasterr) [inline README.md:11]: UltimMC, i changed in notsecrets, and now we are UltimMC launcher

File: README.md [hunk 4]
```diff
@@ -1,31 +1,32 @@
...
+4. Go to account settings, and you will be requested to use email and password.
+5. As your email use your username, and then type a random password or use ElyBy account.
+6. Save it.
+7. Now enjoy the Launcher.
+
+In case if you are using MacOS/Linux then additional step is to make a script `DevLauncher` executable by using the command `chmod +x DevLauncher` in the terminal
```
Comment (maximmasterr) [inline README.md:17]: UltimMC

**Review by maximmasterr — CHANGES_REQUESTED** [review]

**Comment by Samisafool** [comment]
Huh. There is a PR https://github.com/MultiMC/Launcher/pull/4221 on MultiMC's main repo about exactly this.

**Comment by Samisafool** [comment]
Huh. There is a PR https://github.com/MultiMC/Launcher/pull 4221 on MultiMC's main repo about exactly this.

I doubt it'll be merged as it does not follow the normal contributing rules and also has bad english. Also removes the crowdin link.

**Comment by WesleyBryie** [comment]
> Huh. There is a PR https://github.com/MultiMC/Launcher/pull 4221 on MultiMC's main repo about exactly this.
> 
> I doubt it'll be merged as it does not follow the normal contributing rules and also has bad english. Also removes the crowdin link.

I beg your pardon. I only deleted text, not added. And the only other changes were changing "MultiMC" to "Launcher"

**Comment by Samisafool** [comment]
> We keep Launcher open source because

for example. This would be better as 

> We keep the launcher open source because

and

>  If you want to build Launcher yourself, check

would be better as 

>  If you want to build the launcher yourself, check

Also you removed the crowdin link.

**Pull request merged**